src/test/EventEmitter.test.js

```diff
@@ -13,36 +14,42 @@ describe('Test EventEmitter module about', () => {
     eventHandler1 = jest.fn();
   });
 
-  test('A handler should subscribe event using on method', () => {
+  it('should subscribe event using on method', () => {
     eventEmitter.on('event1', eventHandler1);
     expect(eventEmitter.eventHandler.event1.has(eventHandler1)).toBe(true);
   });
 
-  test('Many handler should subscribe event using on method', () => {
+  it('many handler should subscribe event using on method', () => {
     const eventHandler2 = jest.fn();
 
     eventEmitter.on('event1', eventHandler1);
     eventEmitter.on('event1', eventHandler2);
 
-    expect(eventEmitter.eventHandler.event1.has(eventHandler1)).toBe(true);
-    expect(eventEmitter.eventHandler.event1.has(eventHandler2)).toBe(true);
+    const eventHandler = eventEmitter.eventHandler;
+
+    expect(eventHandler.event1.has(eventHandler1)).toBe(true);
+    expect(eventHandler.event1.has(eventHandler2)).toBe(true);
```

Comment:
```suggestion
    expect(eventHandler.event1.has(eventHandler1)).toBeTruthy();
    expect(eventHandler.event1.has(eventHandler2)).toBeTruthy();
```
다양한 matcher를 활용해보세요~